docs/section19/19-3.md

```diff
@@ -0,0 +1,28 @@
+# Section 19 - Rule 19-3     
+**Schema Version:** 0.0.29
+**Mandatory Rule:** False    
+**Rule ID:** 19-3     
+**Rule Description:** Weather conditions used in sizing runs to determine baseline equipment capacities shall be based either on design days developed using 99.6% heating design temperatures and 1% dry-bulb and 1% wet-bulb cooling design temperatures.    
+**Rule Assertion:** Options are UNDETERMINED     
+**Appendix G Section:** G3.1.2.2.1      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each RulesetProjectDescription
+
+**Applicability Checks:** None  
+
+**Function Calls:**  None
+
+## Rule Logic:
+- Get the weather for the ruleset project description: `weather = RulesetProjectDescription.weather`
+
+- **Rule Assertion:**
+- Case 1: Heating/cooling design day types do not match the definitions in 90.1-2019; outcome=FAIL: `if (weather.cooling_design_day_type != COOLING_1_0) or (weather.heating_design_day_type != HEATING_99_6): FAIL`
```

Comment:
@JacksonJ-KC I think this should also check that these are not equal to null as well. If they are equal to null then i recommend triggering case 2. I am just thinking that software tools like eQuest would have no way of populating this data element in the schema and so would always result in a FAIL with the current logic.